### DISCUSSION TITLE
Export to MP3 option no longer available

### DISCUSSION THREAD

**Discussion by DDGrim** [audacity/audacity#6501] [Q&A]
### Bug description

There has always been the option to export to MP3.  The option is no longer available in the drop down menu when exporting.  I did not make any adjustments in any settings that would effect this problem. 

### Steps to reproduce

1.  Project saved and ready to export
2. Go to “file” “Export Audio”
3. Drop down menu no longer has MP3 option to choose from. 

### Expected behavior

I should be able to export to MP3. I’m using Windows 10 and Audacity 3.5.1

### Actual behavior

WAV, ogg, and other uncompressed files only available. 

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

**Comment by LWinterberg** [comment]
Make sure that mod-mp3 is enabled in Preferences > Modules. If it's missing, try to install Audacity into a different folder - at the moment, we know that a $ in the path breaks module detection (https://github.com/audacity/audacity/issues/6448)
